ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -740,15 +970,22 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
+- as: vsphere-upi-disconnecting-network-f28
```

Comment:
This job does not need upgrade ci, https://redhat-internal.slack.com/archives/CF8SMALS1/p1713842716475199?thread_ts=1713841950.285019&cid=CF8SMALS1

Reply:
Hi @JianLi-RH I understand that we have a large amount of jobs and we can't remember everything. To avoid this happen again, would it be possible to update your tool with skipped features **timely** or track them in a separate doc **timely**?

Reply:
@shellyyang1989 Updated the tool in pr https://github.com/openshift/release/pull/55014

Reply:
Thank you Jian Li!